tests/pytests/unit/modules/test_git.py

```diff
@@ -0,0 +1,198 @@
+"""
+    :codeauthor: Erik Johnson <erik@saltstack.com>
+"""
+
+
+import copy
+import logging
+import os
+import subprocess
+
+import pytest
+import salt.modules.git as git_mod  # Don't potentially shadow GitPython
+from salt.utils.versions import LooseVersion
+from tests.support.mock import MagicMock, Mock, patch
+
+log = logging.getLogger(__name__)
+
+WORKTREE_ROOT = "/tmp/salt-tests-tmpdir/main"
+WORKTREE_INFO = {
```

Comment:
Please do not use mutable types at the global/module level. Documentation here to explain:
https://docs.saltproject.io/en/master/topics/development/tests/index.html#module-global-level-variables